src/components/DebtPool/DebtPool.tsx

```diff
@@ -70,6 +86,9 @@ export const DebtPool: React.FC<IProps> = ({ title, subTitle, data }) => {
                 }}
                 onMouseLeave={event => {
                   const variable: string = event.id.toString()
+                  setLabel('Total Dept')
```

Comment:
it should be "Debt"

Reply:
and you should remove this box-shadow: 
![obraz](https://user-images.githubusercontent.com/48929607/133962310-85bf3257-e685-4401-a17e-4dfe51e5bf80.png)

Reply:
and I think that the amount should be bold, maybe ask Agnieszka about that